ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -132,6 +133,48 @@ public void onPageStarted(WebView webView, String url, Bitmap favicon) {
 
     @Override
     public boolean shouldOverrideUrlLoading(WebView view, String url) {
+        ReadableMap[] policies = ((ReactWebView) view).getNavigationBlockingPolicies();
```

Comment:
Please use 2 spaces for indentation to be consistent with the rest of the codebase.